src/models/Pools.ts

```diff
@@ -20,3 +20,42 @@ export class SovrynPoolSchema {
   @prop()
   lastUpdatedAt?: Date;
 }
+
+@index({chainId: 1, protocol: 1, token0: 1, token1: 1})
+@index({chainId: 1, fee: 1, token0: 1, token1: 1, address: 1}, {unique: true})
+@index({chainId: 1, address: 1})
+@modelOptions({options: {allowMixed: Severity.ALLOW}})
+export class UniswapV3Pool {
+  @prop({required: true})
+  chainId!: string;
+
+  @prop({required: true})
+  protocol!: string;
+
+  @prop({required: true})
+  token0!: string;
+
+  @prop({required: true})
+  token1!: string;
+
+  @prop({required: true})
+  fee!: number;
+
+  @prop({required: true})
+  address!: string;
+
+  @prop()
+  liquidityActive!: number;
```

Comment:
number is too small to handle liquidity, unless you using floating points?

Reply:
let's do string

Reply:
This is Done, check the `src/models/UniswapV3Pool.ts`
I just changed the liquidityActive to string, not sure if need for liquidityLockedToken0 and liquidityLockedToken1 too... Let me know
the current value saved:
```
liquidityActive: "453142182697369458128"
liquidityLockedToken0: 0.00007741645813868914
liquidityLockedToken1: 24035.110619327985
```

Reply:
if mongo can not store floating points, let's trunc floating point and just same integer, it is enought

Reply:
It stores floating points, not sure what is the best to be honest. I think we can keep it but I can change if you say it is better

Reply:
Transforming that into floating-point may loose some precision. I would keep strings.

Reply:
We are good here in that case? Or I missed something?

Reply:
if values you store are tne same type as SDK returning it, then we good and you can resolve this comment.

Reply:
liquidityActive is a JSBI, also yesterday it returned to me a BigNumber, when using the liquidity not from the fullpool object. So I think we are good keeping it as string.
The liquidityLockedToken0 and liquidityLockedToken1 are floats.
So I think it is good right?